packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -308,6 +318,23 @@ const renderApp = (args) => {
     if (error && isRemote()) {
         delete error.stack
     }
+    performance.mark(PERFORMANCE_MARKS.totalEnd)
+
+    let performanceMetrics = {}
+    try {
+        performanceMetrics = getPerformanceMetrics()
+
+        // DISCUSSION! HOW DO WE WANT TO CONTROL THIS?
+        const TBD_FEATURE_FLAG = true
+        if (TBD_FEATURE_FLAG) {
+            // TODO: prettify print
+            console.debug('Performance metrics', performanceMetrics)
+        }
```

Comment:
@adamraya Could you guide me on what's the proper way to add debug logs? How can I leverage your SDK logging proposal to conditionally add logs like this one?